app/assess/templates/macros/scored_section.html

```diff
@@ -0,0 +1,52 @@
+{%- from "govuk_frontend_jinja/components/table/macro.html" import govukTable -%}
```

Comment:
A bit of a weird one to wrap the head around, but technically we don't have scored sections. We have unscored sections and scored criteria, like so:
scored_criteria -> sub_criteria -> theme -> answers
unscored_sections -> sub_criteria -> theme -> answers

Reply:
Hey, thanks for clarifying that for me this morning.  It's a lot more clear now...

 I've pushed https://github.com/communitiesuk/funding-service-design-assessment/pull/58/commits/eb884c180a63acf4e781fbfbbc8eba0d863659c6 to implement the changes we spoke about, if you have any other suggestions please let me know! 👍